.github/workflows/release-tagger.yml

```
@@ -0,0 +1,37 @@
+name: Release Tagger
+description: Automatically creates and pushes a tag when a pull request is opened or edited on the 'dev' branch if the branch name matches a version pattern.
```

actionlint (1.7.4)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)